### PR TITLE
Add recipe for conda package

### PR DESCRIPTION
**Pull request by jordiandreu**
I would like to package the library in conda. To do so, I have added a recipe to build the corresponding conda package.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,11 +1,16 @@
 #!/usr/bin/env python
 # 	-*- coding: utf-8 -*-
 
-from distutils.core import setup
+#from distutils.core import setup
+from setuptools import setup
+
+requirements = ['setuptools', 'python']
```
Comment (jordiandreu) [inline setup.py:7]: The cv module is also a dependency that need to be added.

**Review by jordiandreu — COMMENTED** [review]
I will add the dependency and test again.

**Comment by beteva** [comment]
Good initiative, thanks a lot.
From the front page it seems that you also need opencv >= 2.4
@olofsvensson, could you, please, check if this is still true.

**Comment by jordiandreu** [comment]
I have already found the dependency. As soon I have fixed the package, I'll upload again.

**Comment by jordiandreu** [comment]
I will close this pull request since the recipe is not complete and some more dependencies need to be added. I will reopen it once the missing dependencies are properly included.

**Comment by jordiandreu** [comment]
I have fixed the dependencies in the setup.py file. The conda recipe works fine. The package works for python>=2.7. I have only make some basic tests so,  any feedback  will be welcome.

**Pull request merged**